### DISCUSSION TITLE
Use wagmi outside of components

### DISCUSSION THREAD

**Discussion by feloot** [unicape/use-wagmi#66] [Q&A]
Hi,

First of all thank you very much for putting this package together.

I'm currently migrating my app and couldn't find what I'm looking for in the examples so, could you please describe what is the correct way for using wagmi outside of components ? Since the config is injected in the components tree but does not exist outside of it.

Thanks !

**Comment by unicape** [comment]
Are you saying you want to use it outside of setup?

You can inject USE_WAGMI yourself but that means you need to manage the state yourself
